projects/local-ci/testing/config.yaml

```diff
@@ -41,6 +41,49 @@ workloads:
         command: pipelines test cleanup_cluster_ci
         alway_run: true
 
+  pipelines-light:
+    command_group: rhods
+    test_args:
+    - light
+    steps:
+      pre_cleanup_ns:
+        command: pipelines test cleanup_sutest_ns
+      test:
+        command: pipelines test test_ci
+      post_cleanup_ns:
+        command: pipelines test cleanup_sutest_ns
+        alway_run: true
+
+  pipelines-stress:
+    command_group: rhods
+    test_args:
+    - stress
```

Comment:
@drewrip , you should remove the `test_args: [stress]` field from here, and pass it via the command-line, eg:
```
 --test_args "[e2e_perf,\"10min\"]" 
```
so that you can collapse these three configurations into a single one

Reply:
Sure!